tests/autocausality/test_scoring.py

```diff
@@ -28,12 +28,13 @@ def simple_model_run(rscorer=False):
         if rscorer=False:
             input parameters for metrics functions (such as qini_make_score
     """
-    data_df = synth_ihdp()
-    treatment = "treatment"
-    targets = ["y_factual"]
+    data = synth_ihdp()
+    data_df = data.data
+    treatment = data.treatment
+    targets = data.outcomes
     data_df, features_X, features_W = preprocess_dataset(data_df, treatment, targets)
     # data_df = data_df.drop(columns = ["random"])
-    outcome = targets[0]
+    outcome = targets[0] if type(targets) is list else targets
```

Comment:
In general, `targets` could also be a tuple, for example, or np.ndarray of strings. It would be safer to instead check whether it's a str, and else treat it a sequence.
But in any case, here you know exactly what it's going to be from synth_ihdp(), so why the if-statement?